types/smb2/smb2-tests.ts

```diff
@@ -0,0 +1,4 @@
+function uhOh() {
+	// I don't have any tests!
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/smb2/smb2-tests.ts;
  Line: 2;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.